demos/snackbar.html

```diff
@@ -73,8 +73,7 @@
       <section class="hero">
         <div class="mdc-snackbar mdc-snackbar--active"
              aria-live="assertive"
-             aria-atomic="true"
-             aria-hidden="true">
```

Comment:
**Concern:** client knows deeply about the style implementation (expect aria-hidden to not exists so `visibility: hidden` does not apply).